public/view/overlay-controls/selected-admins-info.jsx

```diff
@@ -22,6 +27,7 @@ var SelectedAdminsInfo = React.createClass({
       <h3>{admin.name}</h3>
       <div>Area: {admin.geo_area_sqkm} km²</div>
       <div>Case data: {this.create_case_data(admin)}</div>
+      <div>Population: {this.commify(admin.population)}</div>
```

Comment:
Decided to go with "Population: No data." option. 

Eventually there will could be many fields without values. Not sure yet how that will be handled.